### DISCUSSION TITLE
Has the use of ASReview in systematic reviews been validated?

### DISCUSSION THREAD

**Discussion by LHofstee** [asreview/asreview#556] [Q&A] [answered]
I would like to know whether the use of ASReview in systematic reviews is validated.

**Comment by Rensvandeschoot** [comment, marked as answer]
The performance of active learning as is implemented in ASReview has been tested in several simulation studies. The very [first results ](https://doi.org/10.1080/00273171.2020.1853501) showed that both the Bayesian and logistic regression classifiers found more than 80% of relevant publications after screening only 10% of all publications. In our publication in [Nature Machine Intelligence](
https://doi.org/10.1038/s42256-020-00287-7) we computed, for one specific model (i.e. Naïve Bayes with TF-IDF), the work saved over sampling (WSS), which is the percentage reduction in the number of records needed to screen achieved by using active learning instead of screening records at random. The WSS at 95% recall for four PRISMA based systematic reviews ranged from 67% to 92%. In an [extensive simulation study]( https://doi.org/10.31219/osf.io/w6qbg) we tested seven different models for another six PRISMA based systematic reviews and  the WSS@95 ranged between 65%-92%.
